src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -187,5 +195,10 @@ public static FollowUpVo getCreateFollowUpData(FollowUp followUp, FollowUpVo fol
         return followUpVo;
     }
 
+    public static LocalityVo getCreateLocalityVo(Inquiry inquiry, InquiryVo inquiryVo) {
+        Locality locality = inquiry.getInquiryAddress().getLocation();
+        return null;
```

Comment:
Why return null from here?

Reply:
sir this is dummy method use for testing i forget to remove it.Remove in commit 4ab6df7